server/src/ingestion.rs

```diff
@@ -0,0 +1,135 @@
+/* Copyright 2022 The MiniModelarDB Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! Support for data ingestion from an MQTT broker.
+//!
+//! To use the ingestor, run the "ingestor.start()" method. This method first creates the client. Then
```

Comment:
Maybe `ingestor.start()` instead of "ingestor.start()".

Reply:
Fixed.